class.photon.php

```diff
@@ -388,6 +389,24 @@ public static function filter_the_galleries( $galleries ) {
 		return $galleries;
 	}
 
+
+	/**
+	 * Runs the image widget through photon.
+	 *
+	 * @param array $instance Image widget instance data.
+	 * @return array
+	 */
+	function filter_the_image_widget( $instance ) {
+		if ( Jetpack::is_module_active( 'photon' ) && ! $instance['attachment_id'] && $instance['url'] ) {
+			jetpack_photon_url( $instance['url'], array(
```

Comment:
This should use something like what is used when [filtering the Text widget](https://github.com/Automattic/jetpack/blob/master/functions.photon.php#L327-L329) because that way it defaults to use the `resize` mode which is more reliable than using this function.
I experienced that while [previously adapting the Image widget to use Photon](https://github.com/Automattic/jetpack/pull/6630/files).
You just need to build the `<img />` with the URL and the `width` and `height` parameters and run it through the `Jetpack_Photon::filter_the_content` method.